rfcs/text/0000-data-processing-robustness.md

```diff
@@ -0,0 +1,75 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Name
+
+DCP data processing robustness principles
```

Comment:
The template says _Replace "RFC Name" above with the name of your RFC. Keep it short and descriptive._ Not sure how to make that  instruction more clear.

Reply:
Probably putting the part to replace in italics.  or just realize that programmers, oddly, are bad at following instructions.